packages/contracts/contracts/LQTY/LQTYStaking.sol

```diff
@@ -7,10 +7,10 @@ import "../Dependencies/SafeMath.sol";
 import "../Dependencies/Ownable.sol";
 import "../Dependencies/CheckContract.sol";
 import "../Dependencies/console.sol";
-import "../Interfaces/ILQTYToken.sol";
 import "../Interfaces/ILQTYStaking.sol";
```

Comment:
Is this for a future patch to be removed?

Reply:
ILQTYStaking will get renamed in a future PR, atm the LQTYStaking just collects the fees, which will get changed to PCV or something similar